packages/bot-web-ui/src/types/root-stores.types.ts

```diff
@@ -6,4 +6,7 @@ import type { TCoreStores } from '@deriv/stores/types';
 export type TRootStore = TCoreStores & {
     gtm?: Record<string, unknown>;
     portfolio?: Record<string, unknown>;
+    ui: TCoreStores['ui'] & {
+        is_onscreen_keyboard_active?: boolean;
```

Comment:
@shafin-deriv Why won't you add it in `packages/stores/types.ts`? 🤔

Reply:
We had a talk about the gtm and portfolio previously. Like not adding them right now as others may not be using them. So I thought the same applies to the UI as well or should I add it to the store package?

Reply:
@shafin-deriv Would be better to update the main type file, For example for GTM @yaswanth-deriv already added some type for them in #8275 so once that card is merge you will have the types too.

Reply:
Got it!! Thanks!